src/components/DailyGrowth/DailyGrowth.jsx

```diff
@@ -172,30 +224,35 @@ export default function DailyGrowth() {
                     }}
                     cursor={{ fill: theme.colors.backgroundTertiary }}
                   />
-                  <Bar
-                    name={t('confirmedCasesShort')}
-                    stackId={STACK_PER_DATE_ID}
-                    dataKey={DAILY_CASES_KEY}
-                    fill={theme.colors.negative}
-                  />
-                  <Bar
-                    name={t('deaths')}
-                    stackId={STACK_PER_DATE_ID}
-                    dataKey={DAILY_DEATH_KEY}
-                    fill={theme.colors.primary}
-                  />
-                  <Bar
-                    name={t('cured')}
-                    stackId={STACK_PER_DATE_ID}
-                    dataKey={DAILY_CURES_KEY}
-                    fill={theme.colors.positive}
-                  />
+
+                  {barGraph.filter(filterGroups).map((item) => (
+                    <Bar
+                      name={item.name}
+                      stackId={STACK_PER_DATE_ID}
+                      dataKey={item.key}
+                      fill={item.color}
+                    />
+                  ))}
                 </BarChart>
               </ResponsiveContainer>
             </StyledBody>
           </ResponsiveTab>
         </Tabs>
       )}
+      <div className='daily-growth-filter'>
```

Comment:
We're using Styletron and the styles defined for this element should be written like that:
```suggestion
      <div
        className={css({
          display: flex,
          justify-content: space-evenly,
        })}
      >
```
Note: `css` is not defined as for now. You should add its definition in line 79:
```js
  const [css, theme] = useStyletron();
```